modules/ROOT/pages/index.adoc

```diff
@@ -1,12 +1,17 @@
 = What's New
 
 == Highlights
+Two Java build options are available at this release:
 
-=== Custom Conflict Resolution
+*  Java Android - enabling development and deployment of Java apps in an android environment (Read more: xref:java-android.adoc[Java Android])
+* Java Platform
+ - enabling development and deployment of Java apps in any environment that supports the JVM model.
 
-The Custom Conflict Resolution feature enhances the automatic conflict resolution feature in Couchbase Mobile 2.x to allow applications to use custom logic to select the winner between conflicting revisions of a document.
+=== Java Platform
```

Comment:
This section seems like it's just repeating the the content from the list item that immeaditatly precedes it. My suggestion would be to consolidate the content into just the list, and remove this section.

Reply:
Consolidated as suggested